pkgs/tools/networking/slirp4netns/default.nix

```diff
@@ -1,27 +1,27 @@
-{ stdenv, fetchFromGitHub, autoreconfHook, pkgconfig, glib }:
+{ stdenv, fetchFromGitHub, autoreconfHook, pkgconfig, glib, libcap, libseccomp }:
 
 stdenv.mkDerivation rec {
-  pname = "slirp4netns";
-  version = "0.3.0";
+  name = "slirp4netns-${version}";
+  version = "0.4.1";
```

Comment:
The commit states 0.4.2, but the version here is 0.4.1.

Reply:
Yes, fixed that.